tests/update/patch_sle.pm

```diff
@@ -143,11 +143,38 @@ sub install_patterns {
     my $pcm = 0;
     my @pt_list;
     my @pt_list_un;
-    my @pt_list_in = split(/ /, script_output("zypper pt -i | grep '^i' | awk -F '|' '{print \$2}' | sort -u | xargs"));
+    my @pt_list_in;
```

Comment:
1.Need add more comments describe why need handle output outside of script_output.
2.Better create function to call